action.yml

```diff
@@ -40,14 +40,19 @@ inputs:
   certificate-profile-name:
     description: The Certificate Profile name.
     required: true
+  files:
+    description: A comma separated list of absolute paths to the files being signed. Can be combined with
+                 the files-folder and file-catalog inputs.
+    required: false
   files-folder:
-    description: The folder containing files to be signed. Can be combined with the file-catalog input.
+    description: The folder containing files to be signed. Can be combined with the files and file-catalog inputs.
     required: false
   files-folder-filter:
     description: A comma separated list of file extensions that determines which types of files will
                  be signed in the folder specified by the files-folder input. E.g., 'dll,exe,msix'.
                  Any file type not included in this list will not be signed. If this input is not used,
-                 all files in the folder will be signed.
+                 all files in the folder will be signed. Supports wildcards for matching multiple file
+                 names with a pattern.
```

Comment:
Same as above, with "glob patterns" instead of "wildcards" if globs are support (but maybe they aren't).